sentry_sdk/__init__.pyi

```diff
@@ -0,0 +1,12 @@
+"""
+Type stub file for mypy.
+Since mypy cannot deal with the runtime calculated __all__ variable in __init__.py.
+"""
+from sentry_sdk.hub import Hub, init
+from sentry_sdk.scope import Scope
+from sentry_sdk.transport import Transport, HttpTransport
+from sentry_sdk.client import Client
+
+from sentry_sdk.api import *  # noqa
+
+from sentry_sdk.consts import VERSION  # noqa
```

Comment:
As pointed out in #375, instead of `__init__.pyi`, an alternative to this would be to duplicate contents of the `api_all` list in `__all__` of `__init__.py`, instead of calculating the list at runtime. I'll leave this decision up to the reviewers.

Reply:
yeah please let's do that instead